src/pyuvsim/data/mwa_128T_layout.csv

```diff
@@ -0,0 +1,130 @@
+Name	Number	BeamID	E	        N	        U
```

Comment:
Seems like we have this file in the repo twice? here and in the reference sims folder. Is that what we want?

Reply:
I think that's also true of a bunch of the following files here. I worry that if we have them in twice they can get out of sync...

Reply:
We talked on the telecon and decided that we should put them in the data folder and just point to where they are (with urls) in the reference sim documentation. And we should update the download scripts to get them from the right place. The idea long term is that we'll move the test and ref sim data to another repo and use pooch to get them.